### DISCUSSION TITLE
MacOS M3 v0.6.6 Install Issue

### DISCUSSION THREAD

**Discussion by middlesidetopwise** [publicsamples/Quetzalcoatl#19] [Q&A]
When I try to extract Quetzalcoatl.Mac.VTS3.zip.001 in The Unarchiver, it says the archive file is corrupted. It still allows me to create Quetzalcoatl.pkg, which upon opening says “The installer could not install the software because there was no software found to install”. Around and around we go…. Please help thanks

**Comment by RickTaylor79** [comment]
Did you download the second part of the archive? 

https://github.com/publicsamples/Quetzalcoatl/releases/download/0.6.6/Quetzalcoatl.Mac.AU.VST3.zip.002

**Reply by middlesidetopwise** [reply]
Yes, I downloaded both files. The Unarchiver says “The archive file is incomplete” for the 002 file. Tried opening them both at once and individually.

**Reply by RickTaylor79** [reply]
I don't know.  it works fine here and on another Mac.  Try downloading them again, make sure you have both files in the same folder. If it's still not working try [Keka](https://www.keka.io/en/).

There's a 2gb file limit in Github or I'd provide a single zip..

**Reply by middlesidetopwise** [reply]
Just tried to download both files again after restart, same issues. Oh well 🙁

**Reply by RickTaylor79** [reply]
I'll see about uploading a single zip somewhere else in the morning. it's just weird that this doesn't work for you, can't explain it..

**Comment by middlesidetopwise** [comment]
So I was able to get the plugin working after a software update, but the 002 file still doesn’t open in Unarchiver on my iMac or my partner’s M1 MacBook

**Reply by RickTaylor79** [reply]
Very weird. I don't understand why an update would make a difference, possibly the download got corrupted 'cos of a shaky network connection?  Anyway, here's a single zip file created with the built in Mac OS archiver:

https://drive.google.com/file/d/1odskK9WQ65VcKTG0R3PjRgvSpfS3t_MK/view?usp=share_link

**Comment by middlesidetopwise** [comment]
Ok so here’s what I did. I downloaded Unarchiver to my partner’s MBP, and redownloaded the file. Worked fine, so I took the uncompressed .pkg and put it on my computer with an external drive. Ran a software update on my iMac (was running Sonoma 14.2). Plugin installed just fine.

Will there be two versions if I now try to install this version? Thank you for your efforts and patience, really lovely synth!!

**Reply by RickTaylor79** [reply]
No, it should just replace the older copy. It's exactly same, just packaged slightly differently. 

Thanks! hope you find it useful. Next update should improve performance significantly.
